src/test/java/hudson/plugins/ec2/AmazonEC2CloudTest.java

```diff
@@ -92,6 +95,10 @@ public void testPrivateKeyUpdate() throws Exception {
     }
 
     private HtmlForm getConfigForm() throws IOException, SAXException {
-        return r.createWebClient().goTo("configure").getFormByName("config");
+        if (Jenkins.getVersion().isNewerThanOrEqualTo(new VersionNumber("2.205"))) {
+            return r.createWebClient().goTo("configureClouds").getFormByName("config");
+        } else {
+            return r.createWebClient().goTo("configure").getFormByName("config");
+        }
```

Comment:
:open_mouth:

Reply:
Its the rework to the clouds section so I added this code to be forwards compatible

Reply:
yes I've checked during my review because I wanted to be sure of understanding it. But it shocked me at a first glance